apps/service_providers/tests/test_runnables_cancellation.py

```diff
@@ -40,6 +40,7 @@ def session(fake_llm_service):
 
 @pytest.mark.django_db()
 @patch("apps.service_providers.llm_service.adapters.get_tools")
+@patch("apps.service_providers.llm_service.history_managers.ExperimentHistoryManager.add_messages_to_history", Mock())
 def test_simple_runnable_cancellation(get_tools, session, fake_llm_service):
     get_tools.return_value = []
     runnable = _get_mocked_history_recording(session, SimpleLLMChat)
```

Comment:
is this `_get_mocked_history_recording` utility still useful?

Reply:
Yes! That's the proper place to mock these. Thanks

Reply:
https://github.com/dimagi/open-chat-studio/pull/950/commits/733f0f0a414a30946482328eb566569a2d58a901